docs/marketplace/jenkins/enterprise-store/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Setting Up Appcircle Enterprise Store Plugin
+sidebar_label: Enterprise App Store
+description: Enhance powerful plugin to publish your builds to appcircle app store
+tags:
+  [
+    overview,
+    concepts,
+    app store,
+    internal-testing,
+    beta testing,
+    binary distribution,
+    ipa distribution,
+    apk distribution,
+  ]
+sidebar_position: 2
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+The Appcircle Enterprise Store plugin allows users to publish their apps to appcircle app store
+
+### Install Appcircle Enterprise Store Plugin
+
+Go to your Jenkins dashboard and follow, Manage Jenkins > Plugins and search for "Appcircle Enterprise Store"
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/sp-158-installation_steps.png' />
+
+### Add Plugin in Build Steps
+
+Go to your configuration page of the project and add a build step.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/sp-158-store-add_pannel.png' />
+
+### Configure Plugin
+
+After adding the plugin into your build steps make sure you fulfill required inputs.
+Also, do not forget to add the plugin after your build steps. Because you will be asked to provide the build path.
```

Comment:
After adding the plugin to your build steps, ensure that you provide all required inputs. Additionally, remember to place the plugin after your build steps. You will need to specify the build path during configuration.